script.js

```diff
@@ -7,23 +7,32 @@ function setup() {
 
 
 function makePageForEpisodes(episodeList) {
+  let header = document.getElementById("header")
+  let searchBar = document.createElement("input")
+     searchBar.type = "search"
+     searchBar.placeholder = "Search Now"
+     header.appendChild(searchBar)
+
 
   const rootElem = document.getElementById("root");
+
   for (let i = 0; i < episodeList.length; i++){
   let rect = document.createElement("div")
   //styling of ep divs
-  rect.style.backgroundColor = "indianred";
+  rect.style.backgroundColor = "darkgrey";
   rect.style.border = "2px solid black"
   rect.style.margin = "10px"
   rect.style.width = "360px";
   rect.style.height = "410px"
   rect.style.textAlign = "center"
+  rect.style.justifyContent = "space-evenly"
```

Comment:
I see that you are doing a lot of styling in JS.  Good job on practicing DOM! Is there any other reason why you choose to do styling in JS rather than CSS?